Providers/SwrveProvider.m

```diff
@@ -0,0 +1,38 @@
+//
+//  SegmentioProvider.m
+//
+
+#import "SwrveProvider.h"
+#import "ARAnalyticsProviders.h"
+#import "Swrve.h"
+
+@implementation SwrveProvider
+
+#ifdef AR_SWRVE_EXISTS
+- (id)initWithAppID:(NSString *)appID apiKey:(NSString *)apiKey {
+    [Swrve sharedInstanceWithAppID:appID.intValue apiKey:apiKey];
+    return [super init];
+}
+
+- (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email {
+    if (!userID && !email) return;
+    
+    NSMutableDictionary *traits = NSMutableDictionary.dictionary;
+    if (userID) traits[@"id"] = userID;
```

Comment:
you're already checking for userID & email's existence, may as well not use a mutable dictionary at all here